ansible/tasks/installation.yml

```diff
@@ -2,6 +2,5 @@
 
 - name: install ansible helper packages
   package:
-    name: '{{ item }}'
+    name: '{{ ansible_packages }}'
```

Comment:
Just noticed that we should maybe rename that role to not interfere with the `ansible_*` variable namespace.

Reply:
But this would be another PR.